Updated CONTRIBUTING.md template-tool file

F-said opened the pull request:
- populated containers section
- integrated outline & directory details 
- linked [contributing wiki doc](https://ndclab.github.io/wiki/docs/contributing.html) for workflow since it already comprehensively details the workflow for both internal and external members.

F-said committed:
Integrated CONTRIBUTING files, populated containers

F-said committed:
Linked workflow to wiki

jessb0t reviewed (commented):
Thanks, @F-said !

A couple of questions:

- Roadmap: should the "readme.md" be a link? (I ask because of the link included in the "Containers" section.)
- Directory Structure:

    1. - Is the hyphen in the name necessary?
    2. - Can we create a version that doesn't specify "baseEEG"?

- Should "Scripts" and "Containers" be sub-headings under "Directory Structure"?
- Is the implication in "Scripts" that this is where someone should put newly created Python modules?  If so, I suggest we be explicit about this.
- Containers:

    1. - Typo in "reproducibility"
    2. - The link on the readme seems to link back to this contributing.md file when I click it?

F-said committed:
Updated CONTRIB according to review

F-said commented:
Hi @jessb0t!

made the following changes:

- included a link to the README.md
- Directory structure:
    - yes, internal markdown links break with spaces
    - updated the root folder to be "project-name"
- made the purpose of "scripts" explicit
- corrected container mistakes

jessb0t committed:
Add note to update readme link

F-said committed:
Fixed readme link for containers

F-said committed:
Merge branch 'updateCONTRIB-farukh' of https://github.com/NDCLab/template-tool into updateCONTRIB-farukh

F-said committed:
Swapped README links
